editor.planx.uk/src/theme.ts

```diff
@@ -22,6 +22,7 @@ import { gridClasses } from "@mui/x-data-grid";
 import type {} from "@mui/x-data-grid/themeAugmentation";
 import { TeamTheme } from "@opensystemslab/planx-core/types";
 import { getContrastTextColor } from "styleUtils";
+import { switchClasses } from "ui/shared/Switch";
```

Comment:
We should be able to import this directly and remove the export from out `Switch` by adding the following line - 

```suggestion
// eslint-disable-next-line no-restricted-imports
import { switchClasses } from "@mui/material/Switch";
```